### DISCUSSION TITLE
Clean / Reset: Song Playback Timeline

### DISCUSSION THREAD

**Discussion by scysys** [AzuraCast/AzuraCast#6081] [General]
Any tools on board to clean areas like "Song Playback Timeline"?

Actually i clean it directly over the database.
I wonder if there is an faster way that is already integrated in azura which i not see?

Thanks!

**Comment by BusterNeece** [comment]
@scysys To "clean" it? It's meant to be a fairly authoritative record of what we've seen play on a station. We'd generally recommend exporting it, then scrubbing it from there.

**Reply by scysys** [reply]
Ah sorry. Maybe when i write i truncate the table inside the db its better. Sometimes im not sure how to write correctly in english :)

**Reply by BusterNeece** [reply]
We have a System Setting for how long to keep your history; we automatically will truncate records older than that.

**Reply by scysys** [reply]
Ok, thanks.
